docs/cql/ddl.rst

```diff
@@ -302,6 +302,7 @@ Modifying a keyspace with tablets enabled is possible and doesn't require any sp
 - The ``ALTER`` statement may take longer than the regular query timeout, and even if it times out, it will continue to execute in the background.
 - The replication strategy cannot be modified, as keyspaces with tablets only support ``NetworkTopologyStrategy``.
 - The ``ALTER`` statement will fail if it would make the keyspace :term:`RF-rack-invalid <RF-rack-valid keyspace>`.
+- After RF is increased and the tablets migrations caused by that are finished, client applications should be restarted. Without restart drivers will not know about new replicas, which may cause request imbalance.
```

Comment:
Strictly speaking, those are tablet transitions not migrations. 

The doc should say precisely how the user can tell when this condition is met. Which is after ALTER statement completes successfully.

Missing comma: "Without a restart, drivers will"